### ISSUE TITLE
AutoModel.from_pretrained - Which model is loaded

### ISSUE THREAD

**Issue by MayuraRam** [xenova/transformers.js#849]
### Question

I am using AutoModel.from_pretrained("Xenova/yolos-tiny") to load the Yolos model for object detection. Does transformers.js load the model_quantized.onnx  by default? Would I be able to load model.onnx? 

A related question: Is there a way to check which model is loaded once the model is loaded?

**Comment by xenova** [comment]
Right, by default, Transformers.js uses the 8-bit quantized model (model_quantized.onnx). With Transformers.js v2, you can specify `{ quantized: false }` to use the unquantized (fp32) model:
```js
AutoModel.from_pretrained("Xenova/yolos-tiny", { quantized: false })
```

In Transformers.js v3, this will change to be `dtype`:
```js
AutoModel.from_pretrained("Xenova/yolos-tiny", { dtype: 'q8' }) // or 'fp32' or 'fp16' or ...
```

**Comment by MayuraRam** [comment]
I was trying to analyze the model (in this case - https://huggingface.co/Xenova/detr-resnet-50) , using the python onnx module. The module is not reporting out the quantization type. How (which tool chain) is used to create these models for use with ORT-web and transformers.js? How do I verify the quantization type?

**Comment by gyagp** [comment]
There are 3 models there, model.onnx (fp32), model_fp16 (fp16) and model_quantized (int8). You may open these models with [Netron](https://github.com/lutzroeder/netron), and look into the details. 
For model_fp16, there is a op called cast at the very beginning to cast input data from fp32 to fp16. And in the end, there is another cast to cast output from fp16 to fp32.
For model_quantized, you may see op DynamicQuantizeLinear for quantization, then MatMul is replaced with MatMulInteger.
